packages/@romejs/js-compiler/transforms/lint/unsafeNegation.ts

```diff
@@ -0,0 +1,22 @@
+import {Path} from '@romejs/js-compiler';
+import {AnyNode} from '@romejs/js-ast';
+import {markup} from '@romejs/string-markup';
+
+export default {
+  name: 'unsafeNegation',
+  enter(path: Path): AnyNode {
+    const {node} = path;
+    if (
+      node.type === 'BinaryExpression' &&
+      (node.operator === 'in' || node.operator === 'instanceof') &&
+      node.left.type === 'UnaryExpression' &&
+      node.left.operator === '!'
+    ) {
+      path.context.addNodeDiagnostic(node, {
+        category: 'lint/unsafeNegation',
+        message: markup`Unsafe usage of negation operator in left side of binary expression`,
```

Comment:
You shouldn't need `markup` here since it's only for escaping interpolated values.

```suggestion
        message: 'Unsafe usage of negation operator in left side of binary expression',
```

Reply:
Makes sense.  Done.

Reply:
Makes sense